src/modules/pagination/actions.js

```diff
@@ -48,10 +51,26 @@ function renderNewResults(action) {
                     });
                 }
             }
+            
+            if(action === "firstPage"){
```

Comment:
previously next and prev values have been stored in actions.js file within pagination. So you can move 'firstPage' and 'lastPage' values also inside that file for consistency.